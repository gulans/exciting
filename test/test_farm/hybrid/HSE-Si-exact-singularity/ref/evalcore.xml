<?xml version="1.0" encoding="UTF-8"?>
<evalcore>
  <species is="1" chemicalSymbol="Si">
    <atom ia="1">
      <state ist="1" n="1" l="0" k="0" eigenvalue="-6.519297548003e1"/>
      <state ist="2" n="2" l="0" k="0" eigenvalue="-4.702988436587e0"/>
      <state ist="3" n="2" l="1" k="1" eigenvalue="-3.102923507428e0"/>
      <state ist="4" n="2" l="1" k="1" eigenvalue="-3.079192814085e0"/>
    </atom>
    <atom ia="2">
      <state ist="1" n="1" l="0" k="0" eigenvalue="-6.519297548003e1"/>
      <state ist="2" n="2" l="0" k="0" eigenvalue="-4.702988436587e0"/>
      <state ist="3" n="2" l="1" k="1" eigenvalue="-3.102923507428e0"/>
      <state ist="4" n="2" l="1" k="1" eigenvalue="-3.079192814085e0"/>
    </atom>
  </species>
</evalcore>
