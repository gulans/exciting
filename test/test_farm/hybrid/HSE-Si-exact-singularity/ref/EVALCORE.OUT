 
Species :    1 (Si), atom :    1
 n =  1, l =  0, k =  1 :   -65.19297548    
 n =  2, l =  0, k =  1 :   -4.702988437    
 n =  2, l =  1, k =  1 :   -3.102923507    
 n =  2, l =  1, k =  2 :   -3.079192814    
 
Species :    1 (Si), atom :    2
 n =  1, l =  0, k =  1 :   -65.19297548    
 n =  2, l =  0, k =  1 :   -4.702988437    
 n =  2, l =  1, k =  1 :   -3.102923507    
 n =  2, l =  1, k =  2 :   -3.079192814    
